### PR TITLE
Progress on buttons

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
Depends to PR https://github.com/Samsung/TAU/pull/1570

[Issue] https://github.com/Samsung/TAU/issues/1560
[Problem] Progress on Button needs implemented
[Solution]
  - added new templates

[Screenshot]
image

image

image

### CODE REVIEW

File: examples/mobile/UIComponents/components/progress/index.html [hunk 1]
```diff
@@ -42,6 +42,30 @@ <h1>Progress</h1>
...
+				<a href="bottom-bar-progress-item/progress.html">Item Progress</a>
+			</li>
+			<li class="ui-li-anchor">
+				<a href="progress-buttons.html">Progress on Buttons</a>
+			</li>
+			<li class="ui-li-anchor">
+				<a href="progress-dialog.html">Progress on Dialog</a>
+			</li>
+			<li class="ui-li-anchor">
+				<a href="progress-master-onof.html">Progress on Master On-Off</a>
```
Comment (HunseopJeong) [inline examples/mobile/UIComponents/components/progress/index.html]: Is typo? onoff.

File: examples/mobile/UIComponents/components/progress/progress-master-onof.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```
Comment (HunseopJeong) [inline examples/mobile/UIComponents/components/progress/progress-master-onof.html]: Need to change the file name

File: examples/mobile/UIComponents/components/progress/index.html (hunk 1 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/progress/index.html]: Done

File: examples/mobile/UIComponents/components/progress/progress-master-onof.html (hunk 2 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/progress/progress-master-onof.html]: done.

**Review by lbardeli — APPROVED** [review]

**Pull request merged**